packages/docs/content/components/button.mdx

```diff
@@ -123,17 +145,42 @@ Each button variation (outline, solid, ghost) has an alternate option. The alter
   />
 </ThemeContent>
 
-### Changing the size
+### Button sizes
+
+Buttons can exist in two sizes other than default: "big" or "small".
 
-Buttons can exist in two alternative sizes: "big" or "small". See the example below.
+<ThemeContent onlyThemes={['healthcare']}>
+
+#### Big button (Unused)
```

Comment:
Note that this will have the same problem identified earlier this week of ghost headings in the TOC

Reply:
Thanks for calling that out, I'll try out a different solution for now on this page

Reply:
As I'm looking at this because it's a level 4 heading it will not show up in the TOC anyways to this item is a non-issue for now. However, there are still issues on this page with level 3 headings